node-wasm/src/client.rs

```diff
@@ -409,3 +409,116 @@ impl WasmNodeConfig {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::time::Duration;
+
+    use super::*;
+    use crate::worker::NodeWorker;
```

Comment:
let's move those 2 below other imports

Reply:
both below? In other tests we seem to start with `super`, so I've put that one on top and moved crate import down.